src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -98,14 +93,24 @@ const Contact = ({
   name,
   primaryPerson,
   status,
+  avatar,
   primaryAddress,
   handleSingleConfirm,
   setContactFocus,
 }: Props): ReactElement => {
   const { t } = useTranslation();
   const [newsletter, setNewsletter] = useState(SendNewsletterEnum.None);
+  const [updatingSingle, setUpdatingSingle] = useState(false);
   const { classes } = useStyles();
+  const locale = useLocale();
 
+  const newsletterOptions = [
+    { value: SendNewsletterEnum.Physical, name: t('Physical') },
+    { value: SendNewsletterEnum.Email, name: t('Email') },
+    { value: SendNewsletterEnum.Both, name: t('Both') },
+    { value: SendNewsletterEnum.None, name: t('None') },
+  ];
```

Comment:
You could defined this outside the component.

Reply:
Bill added this. But it needs to be inside the component because it uses `t()`

Reply:
You can use `import i18n from 'src/lib/i18n';` to translate it outside the component